frontend/src/components/Login.tsx

```diff
@@ -1,28 +1,28 @@
 import { useAuth0 } from "@auth0/auth0-react";
-
-import UserNameNav from "./UserNameNav";
+import { GrLogin } from "react-icons/gr";
+import { AvatarImage, Avatar, AvatarFallback } from "./UI/avatar";
 import { Button } from "./UI/button";
 
 function Login() {
-  const { loginWithRedirect, isAuthenticated } = useAuth0();
+  const { loginWithRedirect, isAuthenticated, user } = useAuth0();
 
   const handleLogin = async () => {
```

Comment:
maybe to add error handling to manage any potential errors during the login process.